vertical-stack-in-card.js

```diff
@@ -44,14 +44,21 @@ class VerticalStackInCard extends HTMLElement {
     }
 
     set hass(hass) {
+        if (this._refCards) {
+            this._refCards.forEach((card) => {
+                card.hass = hass;
+            });
+        }
+    }
+
+    connectedCallback() {
         const config = this._config;
         const root = this.shadowRoot;
         let index = 0;
         if (config.title) {
```

Comment:
You can remove this block and all index++ usages and just use forEach on top of this._refCards